endpoints/openrtb2/video_auction.go

```diff
@@ -274,6 +274,17 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 		return
 	}
 
+	ctx = context.WithValue(ctx, exchange.DebugAllowedContextKey, account.DebugAllowed)
```

Comment:
I really don't like the context. Is there a way we can do without this?

Reply:
Refactored. We have this variable in account. To have this in `bidder -> requestBid` I passed it through function param from `HoldAuction`